CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+# These owners will be the default owners for everything in the repo.
+*       @bringg/foundation
```

Comment:
Github doesn;t like this, maybe because it's apublic repo

Reply:
Fixed the issue.. added foundation team to "Teams".